datahub-web-react/src/app/ingestV2/source/utils.ts

```diff
@@ -303,6 +303,112 @@ export const getTotalEntitiesIngested = (result: Partial<ExecutionRequestResult>
     return entityTypeCounts.reduce((total, entityType) => total + entityType.count, 0);
 };
 
+export const getOtherIngestionContents = (executionResult: Partial<ExecutionRequestResult>) => {
```

Comment:
seems like there is a lot of shared business logic between these two methods. i wonder if the code duplication can be reduced.

Reply:
I know there is duplication. But given that we will be changing this a lot given BI tools design is not finalised I opted to keep them separate for now

Reply:
Will set to specific pixels